tools/RAiDER/models/hrrr.py

```diff
@@ -341,6 +365,25 @@ def __init__(self):
             '+proj=stere +ellps=sphere +a=6371229.0 +b=6371229.0 +lat_0=90 +lon_0=225.0 ' +
             '+x_0=0.0 +y_0=0.0 +lat_ts=60.0 +no_defs +type=crs'
         )
+        self.setLevelType('ml')
+
+
+    def setLevelType(self, levelType):
+        '''Set the level type to model levels or pressure levels'''
+        if levelType in ['ml', 'pl']:
+            self._model_level_type = levelType
+        else:
+            raise RuntimeError(f'Level type {levelType} is not recognized')
+
+        if levelType == 'ml':
+            self.__model_levels__()
+        else:
+            raise NotImplementedError('Pressure levels do not go high enough for HRRR.')
+
+
+    def __model_levels__(self):
+        self._levels  = 137
+        self._zlevels = np.flipud(LEVELS_137_HEIGHTS)
```

Comment:
50 or 137? Believe HRRR only has 51 model levels? https://rapidrefresh.noaa.gov/faq/HRRR.faq.html

Reply:
There are 51 sigma layers and 50 computational layers that are made available to the end user ([Table 7](https://journals.ametsoc.org/view/journals/mwre/144/4/mwr-d-15-0242.1.xml)).